src/Helper/Helper_Notices.php

```diff
@@ -225,4 +213,51 @@ private function html( $text, $class = 'updated' ) {
 
 		remove_filter( 'wp_kses_allowed_html', $allow_form_elements );
 	}
+
+	/**
+	 * If the current page belongs to Gravity PDF, add these hooks.
+	 *
+	 * @since 6.4
+	 */
+	public function maybe_remove_non_pdf_messages() {
+		if ( \GPDFAPI::get_misc_class()->is_gfpdf_page() ) {
+			add_action( 'gform_admin_messages', [ $this, 'return_pdf_notice_messages' ] );
+			add_action( 'gform_admin_error_messages', [ $this, 'return_pdf_error_messages' ] );
+		}
+	}
+
+	/**
+	 * Return Gravity PDF error messages and disregard the existing messages.
+	 *
+	 * @param array $messages The message to be displayed
+	 *
+	 * @return array $this->errors
+	 *
+	 * @since 6.4
+	 */
+	public function return_pdf_error_messages( array $messages ): array {
```

Comment:
A better name for this method would be `override_gravityforms_admin_error_messages()`